components/offlight/sources/new-task-done-instant/new-task-done-instant.mjs

```diff
@@ -0,0 +1,55 @@
+import offlight from "../../offlight.app.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  key: "offlight-new-task-done-instant",
+  name: "New Task Done (Instant)",
+  description: "Emit new event when a task is marked as complete.",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    offlight,
+    http: {
+      type: "$.interface.http",
+    },
+    db: "$.service.db",
+  },
+  methods: {
+    emitTask(task) {
+      this.$emit(task, {
```

Comment:
Shouldn't this store ids in the database to avoid emitting duplicates after 50 items? (i.e. the cache size of `dedupe: unique`)

Reply:
Sorry, I misread this - since this is only used in deployment and the run is webhook-based, it shouldn't have duplicate problems.